client/src/app/app-routing.module.ts

```diff
@@ -67,9 +67,10 @@ const routes: Routes = [
   imports: [
     RouterModule.forRoot(routes, {
       useHash: Boolean(history.pushState) === false,
-      scrollPositionRestoration: 'disabled',
```

Comment:
We disabled `scrollPositionRestoration` and `anchorScrolling` because we have our own scroll logic: https://github.com/Chocobozzz/PeerTube/blob/develop/client/src/app/app.component.ts#L119

Reply:
Thanks, I've reported the changes here.